### PR TITLE
Fix create and import wallet flow

### PR DESCRIPTION
**Pull request by KedziaPawel**
## What does this PR do?
It aligns create and import flow with designs

#### *Any scout cleaning/refactoring?*
Divided create and import function to smaller functions to increase the readability of the code

- [ ] Checked on iOS
- [X] Checked on Android

### CODE REVIEW

File: src/screens/CreateWalletScreen.tsx [hunk 1]
```diff
@@ -79,35 +79,53 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
...
         <Text style={styles.boldedText}>{this.props.email}</Text>
       </Text>
+      <Text style={[styles.notificationDescription, styles.note]}>
+        <Text style={styles.boldedText}>{i18n.notifications.noteFirst}</Text>
+        {i18n.notifications.noteSecond}
+      </Text>
     </>
   );
 
+  navigateToSuccesfullNotificationSubscriptionMessage = () => {
```
Comment (marek-siemieniuk-morawski) [inline src/screens/CreateWalletScreen.tsx:89]: I like it 👍 we should more often extract logic to functions with meaningful names

Comment (marek-siemieniuk-morawski) [reply src/screens/CreateWalletScreen.tsx:89]: ...but I'm wondering if it'd be better to place such functions inside functions? I mean, if we have a method invoked on press of the submit button, wouldn't it better to have there:
```
handleOnPress() { 
  const showSuccessMessage = () => { (...) }

  const showFailureMessage = () => { (...) }

  (...)
  
  doSomething({
    onSuccess: showSuccessMessage
    onFailure: showFailureMessage
  });
}
```  

instead:
```
showSuccessMessage() { (...) }

showFailureMessage() { (...) }

handleOnPress() { 
  doSomething({
    onSuccess: showSuccessMessage
    onFailure: showFailureMessage
  });
}
```

maybe it's a small difference, but this code is more clear IMHO - methods of the components should describe behaviour of the component, but functions `showSuccessMessage`  `showFailureMessage` are just helpers for `handleOnPress` in this particular case.

Hopefully, I got my point

**Review by marek-siemieniuk-morawski — APPROVED** [review]

**Comment by marek-siemieniuk-morawski** [comment]
Linter didn't pass

File: src/screens/CreateWalletScreen.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/screens/CreateWalletScreen.tsx:89]: I like the second version more, the problem with the first version is that the function handleOnPress will still be very long which in my opinion would decrease readability

**Comment by KedziaPawel** [comment]
> Linter didn't pass

how is this possible? locally it works correctly, can you check on your machine?

**Comment by marek-siemieniuk-morawski** [comment]
> > Linter didn't pass
> 
> how is this possible? locally it works correctly, can you check on your machine?

Sure, w8

**Comment by KedziaPawel** [comment]
> > > Linter didn't pass
> > 
> > 
> > how is this possible? locally it works correctly, can you check on your machine?
> 
> Sure, w8

done

**Pull request merged**